src/services/listener.service.ts

```diff
@@ -5,14 +5,40 @@ import { DatabaseService } from "./database.service";
 export class ListenerService implements ITopicSubscription {
 
     databaseService: DatabaseService;
+    // Message types present in the regular file upload flow
```

Comment:
We can design this logic with configuration driven.

Reply:
This will become heavy configuration. Better to keep like this or have a naming convention.